web/src/beta/components/Resizable/hooks.ts

```diff
@@ -99,12 +96,14 @@ export default (
     window.removeEventListener("touchend", onResizeEnd);
   }, [onResize, onResizeEnd]);
 
-  unbind = unbindEventListeners;
-
   useEffect(() => {
+    if (!isResizing) {
+      unbindEventListeners();
+      return;
+    }
     bindEventListeners();
```

Comment:
```suggestion
    if (!isResizing) {
      return;
    }
    bindEventListeners();
```
I think this isn't necessary because the events will be unbound in the clean callback.

Reply:
I added PR description. and the reason of this code is `c`.
I or the previous code want to detach events after finish resizing.